### DISCUSSION TITLE
Vaccine Spotter Logo

### DISCUSSION THREAD

**Discussion by WillDosSantos** [GUI/covid-vaccine-finder#168] [Show and tell]
Decided to work on a branding idea. Probably not on the top of the list of priorities but thought it might help! 
![branding-2](https://user-images.githubusercontent.com/1126074/115938014-29c34700-a44e-11eb-88ec-20762ea0ff6f.png)
![branding](https://user-images.githubusercontent.com/1126074/115938017-2a5bdd80-a44e-11eb-983f-c899cd9fa197.png)

**Comment by GUI** [comment]
@WillDosSantos: Oh, I really like this! I know there's been some other favicon work going on (#89, #113, #115, #147), and I've been woefully negligent in following up on those, but I think this might strike the right balance between various things. If you'd be willing to contribute SVG versions (or high-res PNGs), I'd love to use it on the site! I'm torn on which color version I like more, though, so I may just try playing around with both.

Based on some earlier community feedback, there was a note about flipping the magnifying glass around due to the potential weight "Q" carries in the US right now: https://github.com/GUI/covid-vaccine-spotter/issues/89#issuecomment-815866925 I hadn't necessarily had this association myself, but just wondering if you had any thoughts? It seems like this current orientation of a magnifying glass is maybe the [most common for icons](https://thenounproject.com/search/?q=magnify), so I'm not sure if this simplified version would be as immediately recognizable flipped around. But I'm open to suggestions on this front, just wanted to mention it since I had seen that earlier.

Thanks!

**Reply by WillDosSantos** [reply]
@GUI Yea totally! I can make a PR for a favicon with some svgs in there. If you want I can add it to the navbar or something and you can see what you think? Great work on the site BTW! I was able to schedule myself an appointment so much easier with it. Glad to be able to contribute a bit.

**Comment by WillDosSantos** [comment]
@GUI 

[logos.zip](https://github.com/GUI/covid-vaccine-spotter/files/6419087/logos.zip)

Hope all is well. I tried to just create a branch and push the svgs in there but I don't have permission to publish (DUH haha) I got a little ambitious. 
But let me know what you think of those. I added a location pin to the logo. I think it represents the app better but I'm interested in your thoughts.

I also put together a small Figma File for some future styles/ideas if you are interested in collaborating further. I sent an invite to the email you have listed on the site. https://www.figma.com/file/do6W2DOzaeZNbNMrkHz8AT/Vaccine-Spotter-Site-Design?node-id=1%3A3

**Reply by GUI** [reply]
@WillDosSantos: Awesome! Looks great! It might be later this week before I'm able to dig into all this, but thanks so much!

**Reply by WillDosSantos** [reply]
@GUI For sure! If you want to collab or need anything else let me know. I would love to volunteer some time to this :)
